src/zino/tasks/juniperalarmtask.py

```diff
@@ -21,13 +21,19 @@ async def run(self):
         except TypeError:
             return
 
-        device_state.alarms = {
-            "yellow": yellow_alarm_count,
-            "red": red_alarm_count,
-        }
+        if not device_state.alarms:
+            device_state.alarms = {
+                "yellow": 0,
+                "red": 0,
+            }
```

Comment:
Would it maybe make more sense to set default values yellow=0 and red=0 in the DeviceState model itself rather than doing it here?

Reply:
I think since we don't have any other alarm counts (and I can't think of any to follow soon) it is more useful to set it here to not have so much extra unnecessary state information

Reply:
We decided against putting them in as default values, for the time being, namely because of their Juniper-specific nature (our research network is mostly Juniper routers these days, though).